geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/KafkaConsumerFeatureSource.scala

```diff
@@ -46,7 +46,9 @@ abstract class KafkaConsumerFeatureSource(entry: ContentEntry,
     val builder = new SimpleFeatureTypeBuilder()
     builder.init(schema)
     builder.setNamespaceURI(getDataStore.getNamespaceURI)
-    builder.buildFeatureType()
+    val sft = builder.buildFeatureType()
+    schema.getUserData.foreach { case (k, v) => sft.getUserData.put(k, v) }
```

Comment:
(Picky) Would it be cleaner to user SimpleFeatureTypeBuilder.userData before calling build?